samples/monster_generated.h

```diff
@@ -25,6 +25,14 @@ enum Color {
   Color_MAX = Color_Blue
 };
 
+inline Color (&EnumValuesColor())[3] {
```

Comment:
Color* would also cease to function in this c++11 syntax, which requires a valid range:
for (auto c : EnumValuesColor()) {
  std::cout << EnumNameCOLOR_TYPE(c) << std::endl;
}